cmd/lifecycle/cli/flags.go

```diff
@@ -168,6 +168,11 @@ func FlagForceRebase(force *bool) {
 	flagSet.BoolVar(force, "force", *force, "execute rebase even if operation is unsafe")
 }
 
+// FlagInsecureRegistry sets insecure-registry parameter as available
+func FlagInsecureRegistry(insecureRegistry *str.Slice) {
+	flagSet.Var(insecureRegistry, "insecure-registry", "to use 'insecure' registries")
```

Comment:
```suggestion
	flagSet.Var(insecureRegistry, "insecure-registry", "insecure registries")
```
(for parity with `FlagTags` which works similarly)